test/.jshintrc

```diff
@@ -3,6 +3,7 @@
 	"qunit": true,
 
 	"-W053": true,
+	"-W097": true, // disable warning for 'Use the function form of "use strict".'
```

Comment:
See https://github.com/jshint/jshint/issues/741#issuecomment-268786491. If they explicitly support it, so let's keep it. However, it doesn't seem wise to rely on the word of mouth to create exception for the format, unless it's explicitly documented.